[Immich] immich-install: Error / Termination while cleaning-up

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Immich

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/immich.sh)"

### ⚙️ What settings are you using?

- [ ] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📝 Provide a clear and concise description of the issue.

While running the Immich community-sciprts I terminated as shown below:

<img width="781" alt="Image" src="https://github.com/user-attachments/assets/23cdcb37-e3f8-4c78-9b52-ec8dde9b31e4" />

The error seems to be in line 450 of https://github.com/community-scripts/ProxmoxVE/blob/main/install/immich-install.sh as the variable is not defined.

I was able to executed the commands afterwards manually and everything seems to work in general.

### 🔄 Steps to reproduce the issue.

Just executed the standard install script via the Proxmox Shell.

### ❌ Paste the full error output (if available).

See screenshot and here as well:

`[ERROR] in line 1083: exit code 0: while executing command lxc-attach -n "$CTID" -- bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/install/"$var_install".sh)" $?`

### 🖼️ Additional context (optional).

_No response_

--- a/ct/immich.sh
+++ b/ct/immich.sh
@@ -284,7 +284,6 @@ function update_script() {
     msg_ok "Updated ${APP} to v${RELEASE}"
 
     msg_info "Cleaning up"
-    rm -f "$immich_zip"
     $STD apt-get -y autoremove
     $STD apt-get -y autoclean
     msg_ok "Cleaned"

--- a/ct/immich.sh
+++ b/ct/immich.sh
@@ -208,7 +208,7 @@ function update_script() {
       $STD apt install -y ./vchord.deb
       $STD sudo -u postgres psql -d immich -c "ALTER EXTENSION vchord UPDATE;"
       systemctl restart postgresql
-      if [[ ! -f ~/.vchord-version ]] || [[ ! "$(cat ~/.vchord_version)" > "0.3.0" ]]; then
+      if [[ ! -f ~/.vchord_version ]] || [[ ! "$(cat ~/.vchord_version)" > "0.3.0" ]]; then
         $STD sudo -u postgres psql -d immich -c "REINDEX DATABASE;"
       fi
       echo "$VCHORD_RELEASE" >~/.vchord_version
@@ -284,6 +284,7 @@ function update_script() {
     msg_ok "Updated ${APP} to v${RELEASE}"
 
     msg_info "Cleaning up"
+    rm -f "$immich_zip"
     $STD apt-get -y autoremove
     $STD apt-get -y autoclean
     msg_ok "Cleaned"

--- a/install/immich-install.sh
+++ b/install/immich-install.sh
@@ -447,7 +447,6 @@ motd_ssh
 customize
 
 msg_info "Cleaning up"
-rm -f "$tmp_file"
 $STD apt-get -y autoremove
 $STD apt-get -y autoclean
 msg_ok "Cleaned"

